manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "manifest_version": 2,
+  "name": "emoji-everywhere",
+  "description": "Convert unicode emoji to images on all webpages",
+  "version": "0.1.5-beta",
+  "author": "Pierre Rudloff",
+  "developer": {
+    "name": "Pierre Rudloff <contact@rudloff.pro>"
+  },
+  "incognito": "spanning",
+  "bugs": "https://github.com/Rudloff/emoji-everywhere/issues",
+  "homepage_url": "https://addons.mozilla.org/firefox/addon/emoji-everywhere/",
+  "devDependencies": {
```

Comment:
We don't need the following properties in `manifest.json`:
* `bugs`
* `devDependencies`
* `keywords`
* `license`
* `main`
* `repository`